docs/requirements-specification.md

```diff
@@ -55,6 +55,16 @@ _This content item details the use case overview of the system vision in its use
 
 ### UC4: Identify capability gap
 
-### UC5: Find expert in a field
+### UC5: Find an expert in a field
+
+| UC5 | Find an expert in a field
+|---|---|
+| Actors | Practitioners or Students |
+| Pre-conditions | - The system contains a list of researchers with their capabilities. |
+|  |  - The user has a set of capabilities to use for looking up researchers.|
+| End conditions (success) | The user sees a list of matched researchers with their capabilities. |
+| Steps | 1. The user type in a set of capabilities separated by a space. |
```

Comment:
I would say that we are going towards specifying the solution, something that we should not specify as requirements engineers. Furthermore, in this case since no previous step is required, it is implied that we are in the homepage of the software.

Reply:
I would avoid relying on implications. If this functionality is accessible from the main page, then state this in the requirement. That does not impose on the solution space, it is a "need as formulated by one of the stakeholders", i.e., a requirement.

Reply:
This is the solution that I require as a stakeholder; the user immediately starts typing in the capability of the researcher that they are looking for. Adding the same step to ALL user stories would create an unnecessary overhead and lead to inconsistency with other user stories. We are writing requirements for a system, and we are unsure about the best solution for the system, whether a web app, a website, or an Excel sheet. I am not relying on "implications" as we are still developing the requirements.